From 15ddf53697b7e41c24a3ce9e2b83a4251aa8db89 Mon Sep 17 00:00:00 2001
From: Shahed Shaikh <shahed.shaikh@qlogic.com>
Date: Thu, 18 Apr 2013 08:10:06 +0000
Subject: [PATCH 552/858] qlcnic: Fix typo in logs

o Debug logs were not matching with code functionality.
o Changed dev_info to netdev_err

Signed-off-by: Shahed Shaikh <shahed.shaikh@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 7bc27a8cd4db2e6ed71d29b5ceae5b4ee873c9af)

Signed-off-by: Arnaldo Carvalho de Melo <acme@redhat.com>
---
 drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c
index 284d628..edd63f1 100644
--- a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c
+++ b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c
@@ -2786,6 +2786,7 @@ static u64 *qlcnic_83xx_fill_stats(struct qlcnic_adapter *adapter,
 void qlcnic_83xx_get_stats(struct qlcnic_adapter *adapter, u64 *data)
 {
  struct qlcnic_cmd_args cmd;
+ struct net_device *netdev = adapter->netdev;
  int ret = 0;
 
  qlcnic_alloc_mbx_args(&cmd, adapter, QLCNIC_CMD_GET_STATISTICS);
@@ -2795,7 +2796,7 @@ void qlcnic_83xx_get_stats(struct qlcnic_adapter *adapter, u64 *data)
  data = qlcnic_83xx_fill_stats(adapter, &cmd, data,
           QLC_83XX_STAT_TX, &ret);
  if (ret) {
-  dev_info(&adapter->pdev->dev, "Error getting MAC stats\n");
+  netdev_err(netdev, "Error getting Tx stats\n");
   goto out;
  }
  /* Get MAC stats */
@@ -2805,8 +2806,7 @@ void qlcnic_83xx_get_stats(struct qlcnic_adapter *adapter, u64 *data)
  data = qlcnic_83xx_fill_stats(adapter, &cmd, data,
           QLC_83XX_STAT_MAC, &ret);
  if (ret) {
-  dev_info(&adapter->pdev->dev,
-    "Error getting Rx stats\n");
+  netdev_err(netdev, "Error getting MAC stats\n");
   goto out;
  }
  /* Get Rx stats */
@@ -2816,8 +2816,7 @@ void qlcnic_83xx_get_stats(struct qlcnic_adapter *adapter, u64 *data)
  data = qlcnic_83xx_fill_stats(adapter, &cmd, data,
           QLC_83XX_STAT_RX, &ret);
  if (ret)
-  dev_info(&adapter->pdev->dev,
-    "Error getting Tx stats\n");
+  netdev_err(netdev, "Error getting Rx stats\n");
 out:
  qlcnic_free_mbx_args(&cmd);
 }
-- 
1.8.3.1